collection-scripts/gather

```diff
@@ -93,6 +99,7 @@ Options:
   -cl, --ceph-logs          Collect ceph daemon, kernel, journal logs and crash reports
   -ns, --namespaced         Collect namespaced resources
   -cs, --clusterscoped      Collect clusterscoped resources
+  --minimal-crds            Collect only relevant CRDs and CSVs
```

Comment:
how about adding short form for it too?

Reply:
Sure, How about -mr, --minimal-resources?

Reply:
MG always collects CRDs or resources. Let's omit it and call it `-m, minimal`?

Reply:
Done